File: reactive-demo-backend/src/main/java/com/example/reactive/demo/config/PersistenceConfig.java
package com.example.reactive.demo.config;

import com.example.reactive.demo.data.AirlineRepository;
import com.example.reactive.demo.model.Airline;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EnableJpaRepositories(
        basePackageClasses = AirlineRepository.class
)
@EntityScan(
        basePackageClasses = Airline.class
)
public class PersistenceConfig {

}

File: reactive-demo-backend-kotlin/src/main/kotlin/dev/divezone/demo/reactive/config/PersistenceConfig.kt
package dev.divezone.demo.reactive.config

import dev.divezone.demo.reactive.data.AirlineRepository
import dev.divezone.demo.reactive.model.Airline
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@EnableJpaRepositories(basePackageClasses = [AirlineRepository::class])
@EntityScan(basePackageClasses = [Airline::class])
class PersistenceConfig